features/support/generic_helpers.rb

```diff
@@ -100,10 +100,18 @@ def summary_page
   @summary_page ||= SummaryPage.new
 end
 
+def summary_page_error
+  @summary_page ||= SummaryPageError.new
+end
+
 def confirmation_page
   @confirmation_page ||= ConfirmationPage.new
 end
 
+def confirmation_page_refund
+  @confirmation_page ||= ConfirmationPageRefund.new
```

Comment:
The value should be @confirmation_page_refund. This is related how ruby's attribute accessor works.